### PR TITLE
Core/PacketIO: Restrict CMSG_EMOTE/CMSG_STANDSTATECHANGE to only allow emotes/stand states that client can send by itself

### PR DESCRIPTION
**Pull request by Viste**
##### CHANGES PROPOSED:
-  hardcode CMSG_EMOTE/CMSG_STANDSTATECHANGE to only allow emotes/stand states that client can send by itself
##### HOW TO TEST THE CHANGES:
try to use emotes/stand states what you don't have in client

##### Target branch(es):
- [x] Master

### CODE REVIEW

File: src/server/game/Handlers/MiscHandler.cpp [hunk 1]
```diff
@@ -543,13 +543,20 @@ void WorldSession::HandleSetSelectionOpcode(WorldPacket & recv_data)
...
+        case UNIT_STAND_STATE_STAND:
+        case UNIT_STAND_STATE_SIT:
+        case UNIT_STAND_STATE_SLEEP:
+        case UNIT_STAND_STATE_KNEEL:
+            break;
+        default:
+            return;
+    }
+
     _player->SetStandState(animstate);
```
Comment (FrancescoBorzi) [inline src/server/game/Handlers/MiscHandler.cpp]: it would look more logical to move `_player->SetStandState(animstate);` inside the `switch`

Comment (Viste) [reply src/server/game/Handlers/MiscHandler.cpp]: No its must be after switch

Comment (FrancescoBorzi) [reply src/server/game/Handlers/MiscHandler.cpp]: ok for me but, just for my understanding, is it only about codestyle or I'm missing something?

Comment (Viste) [reply src/server/game/Handlers/MiscHandler.cpp]: set the state must handle after we get list of allowed stand states

**Comment by Viste** [comment]
and?

**Pull request merged**